tests/js/sentry-test/utils.tsx

```diff
@@ -1,6 +1,15 @@
 // Taken from https://stackoverflow.com/a/56859650/1015027
 function findTextWithMarkup(contentNode: null | Element, textMatch: string | RegExp) {
-  const hasText = (node: Element) => node.textContent === textMatch;
+  const hasText = (node: Element): boolean => {
+    if (!node.textContent) {
+      return false;
+    }
+    if (typeof textMatch === 'string') {
+      return node.textContent.includes(textMatch);
+    }
+    return textMatch.test(node.textContent ?? '');
```

Comment:
in this case, it would already match the first condition, but I believe the compile was not very happy 🙂

Reply:
@priscilawebdev are you referring to the test call or the nullish coalescing? For the latter, you are right that we can actually remove it and explicitly check textContent !== null on line 4 - I'll do that